ArWikiCats/legacy_bots/circular_dependency/joint_class.py

```
@@ -0,0 +1,92 @@
+from ...helps import logger
+from ...translations import (
+    Nat_mens,
+)
+from ..legacy_resolvers_bots import with_years_bot
+from ..utils import RE1_compile, RE2_compile, RE3_compile
+
+
+class CountryLabelAndTermParent:
+    """ """
+
+    def __init__(self, _resolve_callable: Callable = None) -> None:
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)